seed/seed.json

```diff
@@ -589,7 +593,37 @@
             ],
             "filter": {
                 "min_version": "92.1.30.19",
-                "channel": ["NIGHTLY", "BETA"],
+                "min_os_version": "10.0.17134*",
```

Comment:
looks like this is incorrect, see ```TEST(VersionTest, CompareToWildcardString)```.
https://github.com/chromium/chromium/blob/96.0.4643.1/base/version_unittest.cc#L148-L196

Reply:
Fixed

Reply:
@goodov unfortunately, our converter doesn't support this filter :-/ 
we definitely need better validation for `seed.json`

Reply:
Added support for min_os_version and max_os_version via https://github.com/brave/brave-variations/pull/126